h/templates/includes/annotation_card.html.jinja2

```diff
@@ -1,47 +1,67 @@
-{% macro annotation_card(annotation, request, incontext_link, group=none) -%}
+{#
+  Annotation card component.
 
+  Render an annotation card which has a username, timestamp,
+  an annotation quote, description and a list of tags.
+
+  Usage:
+    {{ annotation_card(presented_annotation, request, tag_link) }}
+
+  :param presented_annotation: The annotation object which includes all the fields to display.
+  :param request: The request param containing the route_url method.
+  :param tag_link: Method which generates a link for a given tag.
```

Comment:
Just for future reference, the Python term for an object which can be called/invoked (such as _tag_link_) is a _callable_ - which can be either a stand-alone function or a method in a class.